src/common/data/games/MM02.ts

```diff
@@ -7,109 +7,294 @@ const game: Game = {
-        { "weapon": WeaponNames_MM1.MegaBuster, "damage": 1 },
-        { "weapon": WeaponNames_MM1.RollingCutter, "damage": 10 },
-        { "weapon": WeaponNames_MM1.SuperArm, "damage": 4 },
-        { "weapon": WeaponNames_MM1.IceSlasher, "damage": 0 },
-        { "weapon": WeaponNames_MM1.HyperBomb, "damage": 2 },
-        { "weapon": WeaponNames_MM1.FireStorm, "damage": 1 },
-        { "weapon": WeaponNames_MM1.ThunderBeam, "damage": 1 }
+        { "weapon": WeaponNames_MM2.MegaBuster, "damage": "4" },
+        { "weapon": WeaponNames_MM2.MetalBlade, "damage": "8" },
+        { "weapon": WeaponNames_MM2.AirShooter, "damage": "0" },
+        { "weapon": WeaponNames_MM2.BubbleLead, "damage": "4" },
+        { "weapon": WeaponNames_MM2.QuickBoomerang, "damage": "0" },
+        { "weapon": WeaponNames_MM2.CrashBomber, "damage": "6" },
+        { "weapon": WeaponNames_MM2.TimeStopper, "damage": "0" },
+        { "weapon": WeaponNames_MM2.AtomicFire, "damage": "4, 12, 12" },
+        { "weapon": WeaponNames_MM2.LeafShield, "damage": "0" }
       ],
       "strategy": "",
       "stage": {
-        "strategy": "It's recommended to have the Super Arm or Thunder Beam before going to this stage in order to obtain the Magent Beam located here."
+        "strategy": ""
+      }
+    },
+    {
+      "name": "Heat Man",
+      "weaponDrop": WeaponNames_MM2.AtomicFire,
+      "weaponEffectiveness": [
+        { "weapon": WeaponNames_MM2.MegaBuster, "damage": "4" },
+        { "weapon": WeaponNames_MM2.MetalBlade, "damage": "2" },
+        { "weapon": WeaponNames_MM2.AirShooter, "damage": "4" },
+        { "weapon": WeaponNames_MM2.BubbleLead, "damage": "12" },
+        { "weapon": WeaponNames_MM2.QuickBoomerang, "damage": "4" },
+        { "weapon": WeaponNames_MM2.CrashBomber, "damage": "+" },
+        { "weapon": WeaponNames_MM2.TimeStopper, "damage": "0" },
+        { "weapon": WeaponNames_MM2.AtomicFire, "damage": "+" },
+        { "weapon": WeaponNames_MM2.LeafShield, "damage": "0" }
+      ],
+      "strategy": "",
+      "stage": {
+        "strategy": ""
+      }
+    },
+    {
+      "name": "Wood Man",
+      "weaponDrop": WeaponNames_MM2.LeafShield,
+      "weaponEffectiveness": [
+        { "weapon": WeaponNames_MM2.MegaBuster, "damage": "2" },
+        { "weapon": WeaponNames_MM2.MetalBlade, "damage": "4" },
+        { "weapon": WeaponNames_MM2.AirShooter, "damage": "8" },
+        { "weapon": WeaponNames_MM2.BubbleLead, "damage": "0" },
+        { "weapon": WeaponNames_MM2.QuickBoomerang, "damage": "0" },
+        { "weapon": WeaponNames_MM2.CrashBomber, "damage": "4" },
+        { "weapon": WeaponNames_MM2.TimeStopper, "damage": "0" },
+        { "weapon": WeaponNames_MM2.AtomicFire, "damage": "2, 6, 28" },
+        { "weapon": WeaponNames_MM2.LeafShield, "damage": "+" }
+      ],
+      "strategy": "",
+      "stage": {
+        "strategy": ""
+      }
+    },
+    {
+      "name": "Mecha Dragon",
+      "weaponDrop": "NONE",
```

Comment:
If we update the "WeaponDrop" property inside the Boss interface to be optional (`weaponDrop?: string`) then we can just omit these "NONE" values or explicity set them to `undefined`. We could use null as well, but I find it's usually easier to just work with undefined since it's covered with the optional ? flag while null is not.